src/Merge/MultiConstraint.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+/**
+ * This file is part of the Composer Merge plugin.
+ *
+ * Copyright (C) 2015 Bryan Davis, Wikimedia Foundation, and contributors
```

Comment:
2015 definitely isn't correct for a new file being added :)

Reply:
Good point, I'll update it in this file. We can update the other files in a separate pull request or a general commit.

Reply:
Yeah, others probably want updating, but like you say, don't need to be part of this commit/PR 👍